federation/pkg/kubefed/init/init_test.go

```diff
@@ -496,7 +496,7 @@ func TestCertsTLS(t *testing.T) {
 
 	for i, tc := range testCases {
 		// Make a copy of the client config before modifying it.
-		// We can't do a regular pointer deref shallow copy because
```

Comment:
This is intentional and a well understood abbreviation. I would leave it as it was before.